Hủaıbaı/assets/minecraft/lang/toa_tqg.json

```diff
@@ -118,8 +118,32 @@
   "disconnect.genericReason": "%s",
   "disconnect.quitting": "Chủfaq shảı lủaq",
   "options.language": "Zủ...",
+  "options.sounds": "Gỉaq ru Lảq...",
```

Comment:
So unfortunately inserting 'ru' here doesn't work like you're expecting, and instead this means "things that are both music and sound". We could do something like 'Baq gỉaq ru/roı baq lảq', but in the spirit of keeping all the button labels succinct and free of noun phrases, I suggest we sidestep the problem by just using 'Lảq…' or 'Hủo…' as the label